components/typewriter.js

```diff
@@ -101,7 +101,7 @@ export default class TypeWriter extends Component {
 
       if (delayMap) {
         delayMap.forEach(({ at, delay }) => {
-          if (at === visibleChars || (currentToken && currentToken.match(at)) ) {
+          if (at === visibleChars || (currentToken && currentToken?.match(at)) ) {
```

Comment:
The `currentToken &&` part should be making sure `currentToken` is truthy. I'm not sure what the existential operator is supposed to catch here.